basic-sql-commands.md

```diff
@@ -40,8 +40,8 @@ SHOW DATABASES;
 +----------+
 | Database |
 +----------+
-| db1      |
-| system   |
+| "db1"    |
```

Comment:
Is it really the case that these show up with quotes?
If it is, then the table below needs them as well (the one with aardvarks etc)